src/compiler/typescript_to_rust/azle_generate/src/candid_type/errors/wrong_enclosed_type.rs

```diff
@@ -16,6 +16,11 @@ pub struct WrongEnclosedType {
     location: Location,
 }
 
+// NOTE: Vec, and Opt are not included in this enum because they have much more
+// flexibility with their enclosing type. They can we any sort of TsType. On the
```

Comment:
Something's off with this second sentence but I'm not sure what it's trying to say.

Reply:
should be use instead of we